src/middlewared/middlewared/plugins/netdata.py

```diff
@@ -37,6 +37,27 @@ def __init__(self, *args, **kwargs):
             data['alarms'][alarm].pop('write_path', None)
         return data
 
+    @accepts()
+    async def ips(self):
```

Comment:
They are urls not IPs.

Also, what is the use case?

Reply:
UI requested to know with configurable binding ip's for netdata, where should they redirect the user to launch netdata.
And yes, i should change that to addresses. I wanted to wait till Monday tod discuss this with you as this has a potential problem with `nginx.conf` as there we direct any request with `/netdata` to localhost which in turn means that whichever addresses nginx is using, netdata will be available on those regardless of the fact if user only wants it on one ip.

Reply:
Sounds like thats not the solution.

They have the IP already from the browse, all they need is a path.

If they dont want to hardcode /netdata/ we can provide a method: `netdata.url_path`

Reply:
Also please request them to create a ticket

Reply:
Makes sense @william-gr. I think we should expose `netdata.url_path` after making sure that UI is okay with that else we can close this. Also i'll make sure in the future to request them to create a ticket first.